### PR TITLE
labeled getter and setter

### LINKED ISSUE #576: Labels of geometric points
Firstly, labels of geometric points cannot be turned on or off within scripts; neither for points created as part of the geometry in createCindy() nor of points created by createpoint(). I.e. the commands
`A.labelled = boolean`
and
`A.labeled = boolean`
don't work.

It is, however, possible to (de-)activate the label of a point within the definition in createCindy(). But, secondly, one has to use the command
`labeled: boolean`
whereas the CindyJS manual suggests to use
`A.labelled = boolean`
in the scripts.

So, first, it should be possible to change whether the label is shown. Second, the commands should be consistent throughout CindyJS. As "labeled" is the American English version, it should be the "correct" one.

### PR DESCRIPTION
**Pull request by strobelm**
This should resolve #576. I'm admitting both AE and BE diction.

### CODE REVIEW

File: src/js/libcs/Accessors.js [hunk 1]
```diff
@@ -219,6 +221,11 @@ Accessor.setField = function(geo, field, value) {
             geo.pinned = value.value;
         }
     }
+    if (field === "labeled" || field === "labelled") {
```
Comment (gagern) [inline src/js/libcs/Accessors.js]: If you have the name mapping in `generalFields`, using a single spelling variant here should suffice, right?

**Comment by gagern** [comment]
To fully address #576, this PR should change the documentation to use AE spelling for the sake of consistency.

File: src/js/libcs/Accessors.js (hunk 1 above)
Comment (strobelm) [inline src/js/libcs/Accessors.js]: If i'm reading the code correctly this only applies to the getter code. We never defined something like `if (Accessor.generalFields[field])` for the setter function which could use the mapping.

**Comment by strobelm** [comment]
I added an example and updated the documentation.

File: examples/labeled.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    // See GeoBasics.js for possible attributes.
+  },
+  geometry: [
+    {name:"A", type:"Free", pos:[0,0], labeled:true}
+    // For allowed types, see GeoOps.js.
+    // For allowed properties, see csinit in GeoBasics.js.
+  ] // End of geometry array.
+});
+
+// Remove all comments after adjusting this template for your use case.
```
Comment (gagern) [inline examples/labeled.html]: Please clean comments.

File: examples/labeled.html [hunk 3]
```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+<title>Cindy JS Example</title>
+<meta charset="UTF-8">
+<link rel="stylesheet" href="../build/js/CindyJS.css">
+<script type="text/javascript" src="../build/js/Cindy.js"></script>
+<script id="csinit" type="text/x-cindyscript">
```
Comment (gagern) [inline examples/labeled.html]: I'd drop this unused script.

File: examples/labeled.html [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+// Initialization code, executed once up front.
+
+</script>
+<script id="csdraw" type="text/x-cindyscript">
+fillcircle((0,0),5, color->(1,1,0), alpha->0.2);
+drawtext((0,1.5), "labeled");
+drawtext((0,-6), "unlabeled");
+drawtext((-8,-8), "labeled:" + A.labelled);
+if(|A.xy| < 5,  A.labelled = true, A.labelled = false);
```
Comment (gagern) [inline examples/labeled.html]: Perhaps we should mix spellings, to ensure both work.

File: src/js/libcs/Accessors.js (hunk 1 above)
Comment (gagern) [inline src/js/libcs/Accessors.js]: Perhaps we should reconsider that approach one day, but that would be a separate PR.

**Comment by strobelm** [comment]
This should address all your remaining points.
